src/cluster_infrastructure.py

```diff
@@ -0,0 +1,74 @@
+import requests
+import json
+
+# Returns a list of dicts containing information about each cluster from the IBM API.
+def get_cluster_info(ibm_cloud_api_key):
+  account_iam_req = requests.post(
+    "https://iam.cloud.ibm.com/identity/token",
+    headers={
+      "Content-Type": "application/x-www-form-urlencoded",
+      "Authorization": "Basic Yng6Yng="
+    },
+    data={
+      "grant_type": "urn:ibm:params:oauth:grant-type:apikey",
+      "response_type": "cloud_iam uaa",
+      "apikey": ibm_cloud_api_key,
+      "uaa_client_id": "cf",
+      "uaa_client_secret": ""
+    }
+  )
+
+  account_iam = json.loads(account_iam_req.text)
+
+  accounts_req = requests.get(
+    "https://accounts.cloud.ibm.com/coe/v2/accounts",
+    headers={
+      "Content-Type": "application/json",
+      "Authorization": "bearer " + account_iam["access_token"],
+      "Accept": "application/json"
+    }
+  )
+
+  accounts = json.loads(accounts_req.text)
+
+  if accounts["total_results"] > 1:
+    print("Handle more than one account!")
+
+  account_guid = accounts['resources'][0]['metadata']['guid']
```

Comment:
We can have account_id in the config file itself, I've used the account id in usage_report as well
I feel we can remove the code above this line